src/controls/Jalousie.ts

```diff
@@ -3,6 +3,25 @@ import { Control } from '../structure-file';
 import { ControlBase, ControlType } from './control-base';
 
 export class Jalousie extends ControlBase {
+    protected positionTarget: number | undefined;
```

Comment:
Is there a reason for using "protected"? if you don't have subclasses, I prefer "private". 
Same below.

Reply:
OK